CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Upgrade `targetSdkVersion` and `compileSdkVersion` to API 31
 * Bump `browser-switch` version to `1.2.0`
+* Fix issue where Venmo app is not detected on Android 12 devices
```

Comment:
Do we think it is worth calling out that this change also includes the fix for PayPal? _Maybe_ a merchant is using our signature verification functionality to detect if PayPal is installed 🤷🏻

Reply:
Yeah in case anyone is using that method I updated the CHANGELOG 👍